tests/sentry/integrations/github/test_integration.py

```diff
@@ -85,6 +86,7 @@ class GitHubIntegrationTest(IntegrationTestCase):
 
     def setUp(self):
         super().setUp()
+        settings.SENTRY_FEATURES["organizations:integrations-codeowners"] = True
```

Comment:
@oioki this seems super hacky

Reply:
feels like we can just enable the flag by default

Reply:
Enabled `organizations:integrations-codeowners` by default in server.py